docker/DOCKER.md

```diff
@@ -77,56 +77,68 @@ For guidance on installation, development, deployment, and administration, see o
 
 Docker can be used to manage and run SurrealDB database instances without the need to install any command-line tools. The SurrealDB docker container contains the full command-line tools for importing and exporting data from a running server, or for running a server itself.
 
-```bash
-docker run --rm --pull always --name surrealdb -p 8000:8000 surrealdb/surrealdb:latest start
-```
-
-For just getting started with a development server running in memory, you can pass the container a basic initialization to set the user and password as root and enable logging.
+For just getting started with a development server running in memory, you can start the container with basic initialization arguments to create a root user with "root" as username and password and enable debug logging.
 
 ```bash
-docker run --rm --pull always --name surrealdb -p 8000:8000 surrealdb/surrealdb:latest start --log trace --user root --pass root memory
+docker run --rm --pull always --name surrealdb -p 8000:8000 surrealdb/surrealdb:latest start --log debug --user root --pass root memory
 ``` 
 
-Access to the surrealdb CLI with:
+You can access the server using the same SurrealDB CLI provided in the image by using the `sql` command:
 
 ```bash
 docker exec -it <container_name> /surreal sql -e http://localhost:8000 -u root -p root --ns test --db test --pretty
 ```
 
 <h2><img height="20" src="https://github.com/surrealdb/surrealdb/blob/main/img/gettingstarted.svg?raw=true">&nbsp;&nbsp;Run using Docker Compose</h2>
 
+The Docker image can be used with the `docker compose` command.
+
+Here is an example of a basic `docker-compose.yml` file.
+
+```yaml
+services:
+  surrealdb:
+    command: start 
+    image: surrealdb/surrealdb:v2
```

Comment:
```suggestion
    image: surrealdb/surrealdb:latest
    pull_policy: always
```
?

Reply:
Good idea! Let's do this for the basic example and add a comment for the advanced example.